src/python/pants/backend/pants_info/explain_options.py

```diff
@@ -50,29 +56,53 @@ def register_options(cls, register):
             help="Specify the format options will be printed.",
         )
 
+
+class ExplainOptions(Goal):
+    subsystem_cls = ExplainOptionsOptions
+
+
+class OptionsExplainer:
```

Comment:
Why not use a dataclass for less verbosity? Does prefixing the fields with `_` matter here?

Reply:
Just that I didn't think of it...